int32 channel_1
int32 channel_2
int32 channel_3
int32 channel_4
int32 channel_5
int32 channel_6
int32 channel_7
